zerver/webhooks/gocd/view.py

```diff
@@ -9,13 +9,26 @@
 from zerver.lib.typed_endpoint import JsonBodyPayload, typed_endpoint
 from zerver.lib.validator import WildValue, check_string
 from zerver.lib.webhooks.common import check_send_webhook_message
+from zerver.lib.webhooks.git import get_short_sha
 from zerver.models import UserProfile
 
-MESSAGE_TEMPLATE = """\
-Author: {}
-Build status: {} {}
-Details: [build log]({})
-Comment: {}"""
+COMMIT_INFO_TEMPLATE = (
+    """Triggered on [`{commit_details}`]({commit_link}) on branch `{branch_name}`."""
+)
+TOPIC_TEMPLATE = "{pipeline} / {stage}"
+
+SCHEDULED_BODY_TEMPLATE = """
+**Pipeline** {pipeline}/{stage}: {status}.
```

Comment:
This isn't a sentence, so let's remove the period at the end of the line. I'd also move the status information ...

`**Pipeline {status}**: {pipeline} / {stage}`

Reply:
Done.